pyston/forms/__init__.py

```diff
@@ -585,6 +586,11 @@ def _update_reverse_related_objects(self, resource, model, parent_inst, field_na
 
 class ISODateTimeField(forms.DateTimeField):
 
+    def to_python(self, value):
+        if value:
+            ISODateTimeValidator()(value)
```

Comment:
So you are overriding some default validation in `DateTimeField`? It is strange that you are not changing any parser logic, does that mean that default validator here will not allow ISO format, yet the field will parse it?

Reply:
because django validation was less strict than this one